packages/svgs/src/scripts/load-icons.ts

```diff
@@ -0,0 +1,93 @@
+import * as path from "path";
+import * as fs from "fs";
+import camelCase from "camelcase";
+import {
+    ICONS_SIZES 
+} from "./constants.ts";
+import { getAllFiles } from "./helper.ts";
+
+interface DataItem {
+    name: string;
+    group: number;
+}
+
+export interface Icon {
+    content: string;
+    group: number;
```

Comment:
rename group to size?

Reply:
hmm ya I suppose we could name it that instead, but it is still kind of a group.

Reply:
since I renamed group to size, I had to rename size in OptimizedIcon to dimensions

Reply:
can't OptimizedIcon use the same prop, the group's type should now be Size no?

Reply:
no Size has a width and height of the icon. We're getting this from the svg. Group was the name of the folder basically. so 16, 24 or 36